system-addon/content-src/components/PreferencesPane/_PreferencesPane.scss

```diff
@@ -5,15 +5,16 @@
     background: $white;
     border-left: $border-secondary;
     box-shadow: $shadow-secondary;
-    min-height: 100%;
+    height: 100%;
     offset-inline-end: 0;
-    padding: 40px;
+    overflow: hidden;
```

Comment:
sorry, why can't you just set `overflow-y: auto` here on `.sidebar` (along with `height: 100%`) ?

Reply:
Oh.. is it to keep the done button visible instead of needing to scroll it in?

Reply:
yeah, i needed an element to anchor the button at the bottom.

Reply:
Hrm. The spec in #3273 was just updated to say the done button is always fixed to the bottom: https://mozilla.invisionapp.com/share/NYDAMWECB#/screens/251367746_Preferences_Spec

In that case, we should be able to style the actions area to be fixed to the bottom and make sure there's enough margin below the last option section to have it scroll into view. All without the additional nesting.